tests/test_ebs.py

```diff
@@ -335,6 +335,100 @@ def test_snapshot_trim(self):
         self.assertEqual(len(resources), 1)
 
 
+class SnapshotSetPermissions(BaseTest):
+    # The precondition to these tests in here is that we have an EBS
+    # snapshot with create volume permissions for accounts
+    # 112233445566 and 665544332211
+
+    def test_reset(self):
+        factory = self.replay_flight_data(
+            "test_ebs_snapshot_set_permissions_reset")
+        p = self.load_policy(
+            {
+                "name": "reset-permissions",
+                "resource": "ebs-snapshot",
+                "actions": ["set-permissions"],
```

Comment:
this should have a filter to verify that the pre-conditions re volume is shared.